addons/io_scene_gltf2/__init__.py

```diff
@@ -1765,6 +1939,8 @@ def register():
         blender_ui.variant_register()
     if bpy.context.preferences.addons['io_scene_gltf2'].preferences.animation_ui is True:
         blender_ui.anim_ui_register()
+    if bpy.context.preferences.addons['io_scene_gltf2'].preferences.gltfpack_path_ui is not False:
```

Comment:
Same comment. Always register, add a check in poll

Reply:
Also makes sense. Had problems with poll first, but that's just me not knowing python at all.